kate/src/com.rs

```diff
@@ -383,20 +417,21 @@ pub fn build_commitments(
 		block.len(),
 	);
 
-	let ext_data_matrix = extend_data_matrix(block_dims, &block)?;
+	let ext_data_matrix = par_extend_data_matrix(block_dims, &block)?;
 	let extended_rows_num = block_dims.rows * EXTENSION_FACTOR;
 
 	info!(target: LOG_TARGET, "Time to prepare {:?}", start.elapsed());
 
-	// construct commitments in parallel
-	if block_dims.cols > MAX_BLOCK_COLUMNS as usize {
-		log::error!(
-			target: LOG_TARGET,
-			"Error on Block dimension {:?}",
-			block_dims
-		);
-	}
-	let public_params = testnet::public_params(MAX_BLOCK_COLUMNS as usize);
+	// if block_dims.cols > MAX_BLOCK_COLUMNS as usize {
```

Comment:
Is this a reminder or leftover which could be removed?

Reply:
yeh it can be removed